gene/gene.go

```diff
@@ -156,17 +178,17 @@ func CdnaPosToCodon(g *Gene, cdnaPos int) (dna.Codon, error) {
 	if cdnaPos < 0 {
 		return answer, errors.New("input cDNA position must be positive")
 	}
-	if cdnaPos > len(g.cdnaSeq)-1 {
+	if cdnaPos > len(g.codingSeq.seq)-1 {
```

Comment:
I am not sure why this changed from cdna to codingSeq.  From the error message, cdna seems like what I would expect.

Reply:
In the previous push I was calling this cDNA when what I really ment was CDS. This change is correct, but I changed the rest of the function to properly refer to this as CodingPosToGenomic.